client/src/app/game/soundButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+import css from "./soundButton.module.css";
+
+function SoundButton() {
+
+	const [soundOn, setSoundOn] = useState<boolean>(localStorage.getItem("soundOn") === "on");
+
+	const handleSoundChange = () => {
+		setSoundOn((value) => {
+			localStorage.setItem("soundOn", value === true ? "off" : "on")
+			return !value
+		})
+	};
+
+	return (
```

Comment:
Can you please enable prettier on save in your editor for this project? Otherwise the first time I do any changes in these files I'll get random formatting changes in code. This is my vscode config (with prettier ext. installed)

```
{
	"[json]": {
		"editor.defaultFormatter": "esbenp.prettier-vscode",
		"editor.formatOnSave": true
	},
	"[jsonc]": {
		"editor.defaultFormatter": "esbenp.prettier-vscode",
		"editor.formatOnSave": true
	},
	"[javascript]": {
		"editor.defaultFormatter": "esbenp.prettier-vscode",
		"editor.formatOnSave": true
	},
	"[typescript]": {
			"editor.defaultFormatter": "esbenp.prettier-vscode",
			"editor.formatOnSave": true
	},
	"[typescriptreact]": {
			"editor.defaultFormatter": "esbenp.prettier-vscode",
			"editor.formatOnSave": true
	},
	"[css]": {
		"editor.defaultFormatter": "esbenp.prettier-vscode",
		"editor.formatOnSave": true
	},
	"typescript.tsdk": "node_modules/typescript/lib"
}
```